app/views/govuk_publishing_components/components/_contextual_sidebar.html.erb

```diff
@@ -1,12 +1,15 @@
-<% navigation = GovukPublishingComponents::Presenters::ContextualNavigation.new(content_item, request.path) %>
+<%
+  active_step_nav_content_id = request.query_parameters['step-by-step-nav']
+  navigation = GovukPublishingComponents::Presenters::ContextualNavigation.new(content_item, request.path, active_step_nav_content_id) 
+%>
 
 <div class="gem-c-contextual-sidebar">
   <% if navigation.content_tagged_to_a_reasonable_number_of_step_by_steps? %>
     <!-- Rendering step by step related items because there are a few but not too many of them -->
     <%= render 'govuk_publishing_components/components/step_by_step_nav_related', links: navigation.step_nav_helper.related_links %>
   <% end %>
-
-  <% if navigation.content_tagged_to_single_step_by_step? %>
+  
+  <% if navigation.content_tagged_to_single_step_by_step? or active_step_nav_content_id %>
```

Comment:
I think that in the "real" world, we would need to split up this condition so that if there is an `active_step_nav_content_id`, both the `step_by_step_nav_related` and `step_by_step_nav` will be rendered?

Reply:
Consider adding a new method to `ContextualNavigation` called `show_expanded_step_by_step_navigation?` that does this check.  If you do that, there will be less code in the view, and the new condition will be easier to test.